engineapi/engineapi/routes/metatx.py

```diff
@@ -414,3 +431,32 @@ async def delete_requests(
         raise EngineHTTPException(status_code=500)
 
     return deleted_requests
+
+
+# @app.post("/requests/{request_id}/complete", tags=["requests"])
+@app.post("/requests/complete", tags=["requests"])
+async def complete_call_request_route(
+    tx_hash: str = Form(...),
+    call_request_id: UUID = Form(...),
+    message=Depends(metatx_sign_header),
```

Comment:
Where is the message being used?

Shouldn't `complete_call_request` take the `message["caller"]` as an argument? Otherwise, how does this prevent some random person from signing a message using their own account and completing some other request?

Reply:
Fixed with commit https://github.com/moonstream-to/api/pull/932/commits/e9c46f7a5beb58f3d9aae9e0247664e67c9da16d

Reply:
Somewhere we should normalize `caller` into a checksum address.

Reply:
It happens in `metatx_sign_header`